docs/guides/sdk/authentication.md

```diff
@@ -0,0 +1,156 @@
+---
+contributors: Kevin Lewis, Tim de Heiden
+description: Learn about all of the ways to control authentication with the Directus SDK
+---
+
+# Directus SDK Authentication
+
+The Directus SDK provides multiple ways to [authenticate your requests](/reference/authentication), by either generating
+an access/refresh tokens through login, or by setting a static token.
+
+## Generate, Use, and Refresh Access Tokens
+
+### Using the `authentication()` Composable
+
```

Comment:
I think it would be good to have a little introduction to what the `authentication()` composable does and that it's purpose is to manage tokens for you.